scripts/safe_template_linter.py

```diff
@@ -1554,7 +1525,7 @@ def visit_Str(self, node):
             node: An AST node.
         """
         # Skips '<' (and '>') in regex named groups. For example, "(?P<group>)".
-        if re.search('[(][?]P<', node.s) is None and re.search('[<>]', node.s) is not None:
+        if re.search('[(][?]P<', node.s) is None and re.search('<', node.s) is not None:
```

Comment:
Reviewers: The search for '>' wasn't turning up any additional HTML, but was just catching a bunch of false negatives.